src/buildblock/ProjDataInfoCylindricalNoArcCorr.cxx

```diff
@@ -417,6 +417,7 @@ find_scanner_coordinates_given_cartesian_coordinates(int& det1, int& det2, int&
   det2 = modulo(round(cyl_coords.p2().psi()/(2.*_PI/num_detectors)), num_detectors);
   ring1 = round(cyl_coords.p1().z()/ring_spacing);
   ring2 = round(cyl_coords.p2().z()/ring_spacing);
+  // ORIGINTODO: ^
```

Comment:
I don't think any changes needed as remains in same coordinate system

Reply:
As above